common/src/main/java/com/wynntils/models/players/GuildModel.java

```diff
@@ -87,12 +108,21 @@ public class GuildModel extends Model {
     // Test in GuildModel_OBJECTIVE_STREAK_PATTERN
     private static final Pattern OBJECTIVE_STREAK_PATTERN = Pattern.compile("^§a- §7Streak: §f(?<streak>\\d+)$");
 
+    // Test in GuildModel_TRIBUTE_PATTERN
+    private static final Pattern TRIBUTE_PATTERN = Pattern.compile(
+            "^§[abef6](?<symbol>[ⒷⒸⓀⒿ]?) ?(?<amount>[+-]\\d+) (Ore|Wood|Fish|Crops?|Emeralds?) per Hour$");
+
     private static final int MEMBERS_SLOT = 0;
     private static final int OBJECTIVES_SLOT = 13;
+    public static final int DIPLOMACY_MENU_SLOT = 26;
+    private static final List<Integer> DIPLOMACY_SLOTS = List.of(2, 3, 4, 5, 6, 7, 8);
```

Comment:
Are these fixed slots, do they change with guild level? Can you screenshot the container for me? (I can't login atm, sorry).

Reply:
![Bildschirmfoto vom 2024-05-26 11-27-03](https://github.com/Wynntils/Artemis/assets/59118042/b7532ae7-020c-41e1-a702-0a7c3f9c9b91)

According to the [wiki](https://wynncraft.wiki.gg/wiki/Guilds#Leveling_Guild) there is a total of 7 ally slots to unlock, so this is an exhaustive list.